dev/devicelab/manifest.yaml

```diff
@@ -700,6 +700,13 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  animated_placeholder_perf:
+    description: >
+      Measures CPU usage of a grid of images that use a FadeInImage with an
```

Comment:
`Measures CPU usage` => `Measures frame build and raster times` as `needsMeasureCpuGpu = false`?

Reply:
Updated the langauge - I think the most important feature of this test is acutally the number of frames it pumps rather than the times.  The reason I put CPU to begin with was because that stands as a proxy for CPU usage, but there's more going on for CPU usage than just building frames in the case we're trying to improve.